docs/rules/newline-per-chained-call.md

```diff
@@ -23,12 +23,29 @@ d3
         42
     ])
     .enter()
-    .append('p')
+    .append("p")
     .text(function (d) {
         return "I'm number " + d + "!";
     });
 ```
 
+Another argument in favor of this style is that it improves the readability of diffs when something in the method chain is changed:
```

Comment:
Let’s make the pair of diff examples parallel with the preceding pair and the usual incorrect/correct order:

// Less clear
// More clear